tests/wildcard.test

```diff
@@ -0,0 +1,7 @@
+---------------------------------------------------------------------
+RUN tests/wildcard.hs --hint=data/wildcard.yaml
+FILE tests/wildcard.hs
+module Spec where
+main = print $ (\ _ -> 'a') ()
+OUTPUT
+No hints
```

Comment:
I'd write this as two tests in the same file - one with `module SpecNo` that still triggers const, That makes sure you didn't just write `False` anywhere but are actually matching the wildcards.